packages/web-twig/src/Resources/components/Breadcrumbs/Breadcrumbs.stories.twig

```diff
@@ -3,9 +3,15 @@
 {% block content %}
 
 {# Breadcrumbs Basic #}
-{% include '@components/Breadcrumbs/stories/BreadcrumbsBasic.twig' %}
+<section class="docs-Section">
+  <h2 class="docs-Heading">Default</h2>
+  {% include '@components/Breadcrumbs/stories/BreadcrumbsBasic.twig' %}
```

Comment:
please rename the file to BreadcrumbsDefault

Reply:
renamed to BreadcrumbsDefault